trussworks/react-uswds
# chore(deps): bump node version

# Summary

I noticed that our `.node-version` file was using version 18.10.0. This PR bumps it to 18.14.2. Also saw the warning to run `npx browserslist@latest --update-db` from caniuse-lite. So I did the update and this PR includes the yarn.lock changes. This seems minimal as it's only used by the inclusion of stylelint in our build process.

```sh
=> Found "caniuse-lite@1.0.30001457"
info Reasons this module exists
   - "stylelint#autoprefixer" depends on it
   - Hoisted from "stylelint#autoprefixer#caniuse-lite"
   - Hoisted from "webpack#browserslist#caniuse-lite"
```

## Related Issues or PRs

None

## How To Test

Ensure that the build still works as expected, let me know if there are other steps since I'm not seeing any.
